bzl_library.bzl

```diff
@@ -24,8 +24,11 @@ StarlarkLibraryInfo = provider(
 )
 
 def _bzl_library_impl(ctx):
-    deps_files = [x.files for x in ctx.attr.deps]
-    all_files = depset(ctx.files.srcs, order = "postorder", transitive = deps_files)
+    all_files = depset(
+        ctx.files.srcs,
```

Comment:
Don't you want direct=ctx.files.srcs?

Reply:
That's not required, the first unnamed parameter is equal to "direct".